index.html

```diff
@@ -13,7 +13,21 @@
 </head>
 
 <body>
+  <footer>
```

Comment:
Usar uma class aqui, já que está colocando estilo.
```suggestion
  <footer class="footer">
```